pysumma/simulation.py

```diff
@@ -311,7 +313,10 @@ def start(self, run_option='local',  run_suffix='pysumma_run', processes=1,
         if not prerun_cmds:
             prerun_cmds = []
         self.run_suffix = run_suffix
-        self._write_configuration(name=run_suffix)
+        if write_config:
+            self._write_configuration(name=run_suffix)
+        if write_manager:
+            self._write_file_manager()
```

Comment:
I'm wondering here if we want to have `_write_configuration` take an argument that's just `write_manager` and then roll `_write_file_manager` into `_write_configuration`, since it's only one line of code.

Reply:
I would suggest keep separate, as a clear writing of the manager only. This to me is a quite a separate action / update then writing the entire config.

Reply:
Fair enough, sounds good to me.